src/coinbase/address.ts

```diff
@@ -79,6 +82,64 @@ export class Address {
     return Balance.fromModelAndAssetId(response.data, assetId).amount;
   }
 
+  /**
+   * Returns the historical balances of the provided asset.
+   *
+   * @param assetId - The asset ID.
+   * @param limit - A limit on the number of objects to be returned. Limit can range between 1 and 100, and the default is 10.
+   * @param page - A cursor for pagination across multiple pages of results. Don\&#39;t include this parameter on the first call. Use the next_page value returned in a previous response to request subsequent results.
+   * @returns The list of historical balance of the asset and next page token.
+   */
+  public async listHistoricalBalances(
+    assetId: string,
+    limit?: number,
+    page?: string,
+  ): Promise<[HistoricalBalance[], string]> {
```

Comment:
Can we use an `interface` type as the function argument similar to https://github.com/coinbase/coinbase-sdk-nodejs/blob/1bc6462051e634ee5c5f1b38bf4bbac14adc02f3/src/coinbase/types.ts#L754